JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt

```diff
@@ -53,9 +54,8 @@ fun PostCardPopular(modifier: Modifier = Modifier.None, post: Post) {
             }) {
                 Column(modifier = cardSize) {
                     val image = post.image ?: imageResource(R.drawable.placeholder_4_3)
-                    Container(modifier = LayoutHeight(100.dp) + LayoutSize.Fill) {
-                        DrawImage(image)
-                    }
+                    val imageModifier = ImagePainter(image).toModifier()
```

Comment:
Why extract to a val?

Reply:
Recommended coding style by eng is to extract Modifiers or modifier chains to a val whenever it looks cleaner (and in general treat them as regular variables).

Long modifier chains are difficult to call inline, and hurt readability. In this case, the number of extra tokens `(name).name()` was too high for me to be comfortable putting it at the end of a chain.

Reply:
Ack